--- build.xml	2009-12-17 11:51:24.585192068 +0100
+++ build.xml.oden	2009-12-17 11:53:15.228217736 +0100
@@ -235,14 +235,16 @@
     </target>
 
     <!-- Update the manifest file with a classpath attribute -->
+    <!--
     <target name="-update-manifest-classpath">
       <manifest file="${manifest.file}" mode="update">
         <attribute name="Class-Path"
                    value="${opengrok.lib.files} lib/derbyclient.jar lib/derby.jar"/>
       </manifest>
     </target>
+    -->
 
-    <target name="-pre-jar" depends="-update-manifest-classpath,-update-build-info"/>
+    <target name="-pre-jar" depends="-update-build-info"/>
 
     <!--
         Copy the jars from lib to dist/lib manually if we don't have
